src/color/border.json

```diff
@@ -66,20 +66,27 @@
         }
       },
       "Focus": {
-        "onLight": {
+        "grey": {
           "value": "{color.base.black-opacity-15.value}",
           "public": true,
           "onLight": true,
-          "usage": "Focus border color on light backgrounds",
+          "usage": "Focus border color for neutral backgrounds",
           "wcag": "n/a"
         },
         "onDark": {
           "value": "{color.base.white.value}",
           "public": true,
           "onDark": true,
-          "usage": "Focus border color on dark background",
+          "usage": "Focus border color for dark backgrounds",
           "wcag": "n/a"
-        }
+        },
+        "onLight": {
+          "value": "{color.base.orca.value}",
```

Comment:
I am having growing concerns about using this 'hidden' legacy token for Auro things. It's not fitting the new patten that has been established. https://github.com/AlaskaAirlines/OrionDesignTokens/blob/master/src/color/base.json#L51-L55, 

Currently being used in two other tokens. 
```
color/icon.json
19-      },
20-      "emphasis": {
21-        "onLight": {
22:          "value": "{color.base.orca.value}",

color/text.json
3-    "text": {
4-      "primary": {
5-        "onLight": {
6:          "value": "{color.base.orca.value}",
```

Maybe this is not work for now, but I am wondering if we should consider a black/gray scale of colors as we do with midnight, atlas, breeze, etc ... 

Maybe as example;  
gray-500: #222222
gray-400: #383838
gray-300: #4e4e4e
gray-200: #646464
gray-100: #7a7a7a

Reply:
LOVEE THIS. Let me play around with the grey concept as a base color. 

I agree, I want to get away from legacy tokens